src/precice/bindings/python_future/test/test_bindings_module.pyx

```diff
@@ -12,7 +12,7 @@ class TestBindings(TestCase):
     """
     Test suite to check correct behaviour of python bindings.
     """
-    
+
```

Comment:
Nothing should be changed here

Reply:
Rolled back in https://github.com/precice/precice/pull/421/commits/7c2ec4b1f06467d37244bf51177e0eccb134d453